deployment/docker/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM alpine
 RUN apk add --no-cache curl tar gzip
 COPY build/samlconf.tar /samlconf.tar
 RUN mkdir -p /samlconf && tar xvf /samlconf.tar
-RUN mv -v /samlconf-1.0-SNAPSHOT/* /samlconf
+RUN mv -v /samlconf-1.1-SNAPSHOT/* /samlconf
```

Comment:
❓ Is there no way for us to use the version property here instead of having it in two places?

Reply:
Three places, really, if it's the README, Dockerfile, and Dependences class. :(

Reply:
Changed the Dockerfile